packages/react-router-dom/rollup.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = function rollup() {
     {
       input: `${SOURCE_DIR}/index.ts`,
       output: {
-        file: `${OUTPUT_DIR}/index.js`,
+        file: `${OUTPUT_DIR}/index.mjs`,
```

Comment:
We have to use the `mjs` extension now to prevent this file being resolved as CJS via the `import` field within `exports`.